CH05/final_navigation_2_declarative/candy_store/lib/candy_shop_navigation_2.dart

```diff
@@ -0,0 +1,301 @@
+import 'package:flutter/material.dart';
+
+void main() {
```

Comment:
Since this file is the only one actually used in the 2.0 example, I think we can delete all of the other unused files from the `final_navigation_2_declarative` folder so that the reader can focus only on the relevant code. I would also rename it to `main.dart` once all of the other files are removed.